Documentation/Flashing.md

```diff
@@ -88,18 +88,15 @@ Check the extension of your firmware, it should be `.RDY` now.
 
 ## Pinecil (Pine64)
 
-The MCU used in the Pinecil supports usb-dfu.
+The MCU used in the Pinecil supports usb-dfu. Reference Pine64 [Wiki](https://wiki.pine64.org/wiki/Pinecil).
 
-There is the nice gui [Pinecil Firmware Updater](https://github.com/pine64/pinecil-firmware-updater/releases). In the event that this tool is unavailable or otherwise does not meet your needs, the below process may be used to update the firmware.
+There is the nice gui [Pinecil Firmware Updater](https://github.com/pine64/pinecil-firmware-updater/releases). It works in several types of OS, i.e. Windows/Mac. If your OS is not currently supported by the Updater or it does not meet your needs, the below manual process may be used to update the firmware.
 
-To enter DFU mode, hold the button at the back of the iron while connecting to usb over usb-c.
+To enter DFU mode, long hold the (-) button at the back of the iron, then connect to usb over usb-c.
 Generally all usb controllers work, but some hubs have issues so avoiding hubs may be best.
 
-Once the device is connected while holding the button, the screen will stay **off**.
-
-This is normal, and then you can use the appropriate tool to load the `.hex` or `.bin` file as per your OS.
-
-You can also refer to the Pine64 [Wiki](https://wiki.pine64.org/wiki/Pinecil).

```

Comment:
We probably want to keep the link back to the wiki

Reply:
it is there but higher up.